cmd/manifest/init.go

```diff
@@ -301,7 +302,8 @@ func createFromCompose(composePath string) (*model.Manifest, error) {
 }
 
 func createNewCompose() (*model.Manifest, error) {
-	oktetoLog.Information("docker-compose helps you define a multicontainer application. Learn how to configure a compose here: https://github.com/compose-spec/compose-spec/blob/master/spec.md")
+	oktetoLog.Information("Docker Compose helps you define a multicontainer application")
+	oktetoLog.Information("Learn how to configure a docker compose file here: https://github.com/compose-spec/compose-spec/blob/master/spec.md")
```

Comment:
nit:

```
Learn how to configure a docker compose manifest here
```

we should probably either use `file` or `manifest` everywhere for consistency

Reply:
I am using okteto manifest and docker-compose file. is it fine? or should we go with manifest/file for both?